src/engine/main/Engine.C

```diff
@@ -3039,7 +3042,7 @@ Engine::GatherData(avtDataObjectWriter_p &writer,
             long long cellCountTotal;
```

Comment:
So, it looks like part of this code had already been updated in the past to use `long long`. So, we had a mix of `int` and `long long` logic. Maybe at that time, the belief was that no single domain would exceed `int` in # of primitives.